16:26:56 scenario.file.loc[Ljava.lang.String;@33f88ab
16:26:56 loading BDD file: D:\Automated Testing\JavaSelenium and Appium\PerfectoTest\.\scenarios\suite1.bdd
16:26:57 total test found: 1
16:26:59 

Executing scenario: SampleTest - Sample Test Scenario
16:26:59 pkg: com.qmetry.qaf.automation.step
16:26:59 pkg: com.perfecto.sample.steps
16:26:59 Found duplicate step to load [] with [lower] prority then []
16:26:59 Invoking Teststep: Given i open application
16:27:18 Completed Teststep: Given i open application
16:27:18 Invoking Teststep: When user enter preoduct name to search box
16:27:30 Completed Teststep: When user enter preoduct name to search box
16:27:30 Invoking Teststep: And user able to see all the list of mobiles
16:27:40 Completed Teststep: And user able to see all the list of mobiles
16:27:40 Invoking Teststep: And user select apple I phone gold
16:27:47 Completed Teststep: And user select apple I phone gold
16:27:47 Invoking Teststep: And user Add the product to the cart
16:27:49 Completed Teststep: And user Add the product to the cart
16:27:49 Invoking Teststep: Then verify product is added to the cart
16:27:50 Failure Teststep: Then verify product is added to the cart
java.lang.AssertionError: product is not display in the cart
	at com.qmetry.qaf.automation.core.QAFTestBase.fail(QAFTestBase.java:256)
	at com.perfecto.sample.steps.StepsBackLog.verify_product_is_added_to_the_cart_aroundBody10(StepsBackLog.java:83)
	at com.perfecto.sample.steps.StepsBackLog$AjcClosure11.run(StepsBackLog.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at com.qmetry.qaf.automation.step.JavaStepReporter.javaTestStep(JavaStepReporter.java:101)
	at com.perfecto.sample.steps.StepsBackLog.verify_product_is_added_to_the_cart(StepsBackLog.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.qmetry.qaf.automation.step.JavaStep.doExecute(JavaStep.java:160)
	at com.qmetry.qaf.automation.step.BaseTestStep.execute(BaseTestStep.java:149)
	at com.qmetry.qaf.automation.step.StringTestStep.execute(StringTestStep.java:134)
	at com.qmetry.qaf.automation.step.client.Scenario.execute(Scenario.java:167)
	at com.qmetry.qaf.automation.step.client.Scenario.scenario(Scenario.java:219)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:781)
	at org.testng.TestRunner.run(TestRunner.java:635)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
16:27:50 Completed Teststep: Then verify product is added to the cart
16:27:50 Competed scenario: SampleTest with status FAILURE
